core/domain/customization_args_util.py

```diff
@@ -123,3 +131,217 @@ def validate_customization_args_and_values(
             # not involved (If they are, can we get sample values for the
             # state context parameters?).
             pass
+
+
+def find_subtitled(
```

Comment:
Name is not clear (what is "subtitled")? Please make it clearer.

Also, "find" implies that it's not doing any actions, which seems at odds with the docstring.

Reply:
Changed to ```apply_conversion_fn_on_content```